fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1368,6 +1368,14 @@ opt_broker ::=
     {:
         RESULT = null;
     :}
+    | KW_WITH KW_BROKER
```

Comment:
KW_WITH HDFS/S3 ?

Reply:
We can infer HDFS or S3  from the uri of the file, no need to add HDFS/S3 here